iceberg/iceberg-handler/src/test/queries/negative/delete_iceberg_vectorized.q

```diff
@@ -0,0 +1,10 @@
+set hive.vectorized.execution.enabled=true;
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
```

Comment:
Why do we set these?

Reply:
If there is a way to skip this check for Iceberg tables, then it would be nice

Reply:
I've tried to address this with this commit: [a7fb7f9](https://github.com/apache/hive/pull/3131/commits/a7fb7f90a2fcc3c69b9e533de35b16eda99e3719)

Reply:
Tests passed. I've moved the txnhandler validation logic into UpdateDeleteSemanticAnalyzer, where we already have the table object in hand